src/coralogix.js

```diff
@@ -49,7 +49,7 @@ export class CoralogixLogger {
             invocationId: extractedFields.request_id || 'n/a',
             functionName: this._funcName,
           },
-          message,
+          message: message.replace(/\n+$/, ''),
```

Comment:
remove linefeeds at the end, only

Reply:
why not all ws? `message.trimEnd()` ?

Reply:
there is `trimEnd`? Every day is a school day 😊